eventually/authentication/models.py

```diff
@@ -79,11 +111,11 @@ def create(first_name, middle_name, last_name, email, password):
                           password=password)
         try:
             user.save()
-        except CustomUser.OperationalError():
-            pass
-        return user
+            return user
+        except django.db.utils.IntegrityError:
+            print('ERR: A user with such an email already exists')
```

Comment:
maybe return `pass`

Reply:
This is just for development. Will be removed at the very end of the development.

Reply:
pass added